### PR TITLE
Oroborus (2.0.20): New Package

### PR DESCRIPTION
**Pull request by AndersonTorres**
Oroborus (named after the self-eating snake) is a minimalistic window manager.

### CODE REVIEW

File: nixos/modules/services/x11/window-managers/oroborus.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.oroborus;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.oroborus.enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/x11/window-managers/oroborus.nix]: Use `mkEnableOption` please.

**Comment by jagajaga** [comment]
Please rename commit's name following our guide in `CONTRIBUTING.md`.

**Comment by AndersonTorres** [comment]
Well, I will study the docs and refactor everything later.

File: nixos/modules/services/x11/window-managers/oroborus.nix (hunk 1 above)
Comment (AndersonTorres) [inline nixos/modules/services/x11/window-managers/oroborus.nix]: There is some documentation about mkEnableOption?

Comment (joachifm) [inline nixos/modules/services/x11/window-managers/oroborus.nix]: See https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L27

File: nixos/modules/services/x11/window-managers/oroborus.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.oroborus;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.oroborus.enable = mkEnableOption {
```
Comment (joachifm) [inline nixos/modules/services/x11/window-managers/oroborus.nix]: The syntax of `mkEnableOption` is `mkEnableOption "service-name"`, it does not take an attrset, it returns one.

File: nixos/modules/services/x11/window-managers/oroborus.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.oroborus;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.oroborus.enable = mkEnableOption "oroborus" { };
```
Comment (joachifm) [inline nixos/modules/services/x11/window-managers/oroborus.nix]: `mkEnableOption "foo"` evaluates to a set, which means you are trying to apply a non-function value here. What you want is simply `mkEnableOption "oroborus";`.

File: pkgs/applications/window-managers/oroborus/default.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  src = fetchurl {
+    url = "http://ftp.debian.org/debian/pool/main/o/oroborus/oroborus_${version}.tar.gz";
+    sha256 = "12bvk8x8rfnymbfbwmdcrd9g8m1zxbcq7rgvfdkjr0gnpi0aa82j";
+  };
+
+  meta = {
+    description = "A minimal X window manager";
+    longDescription = ''
+      Oroborus basically is a really minimalistic window manager, in
+      the sense it doesn't provide some things other window managers provide.
```
Comment (joachifm) [inline pkgs/applications/window-managers/oroborus/default.nix]: I think this long description should either be more specific or removed. That a piece of software is minimalistic implies that it lacks features found in other software of its kind.

Comment (AndersonTorres) [inline pkgs/applications/window-managers/oroborus/default.nix]: Done!

**Pull request merged**